openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx

```diff
@@ -165,6 +167,10 @@ const RequestDescription = () => {
     });
   }, [entityData]);
 
+  if (isEmpty(entityData)) {
```

Comment:
Good to use loading flag

Reply:
@chirag-madlani  we are fetching the data from the utils function and setting the state there . <img width="454" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/d78383c9-74fd-4156-bb5c-e85cb43f7f24">